libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -1133,11 +1133,34 @@ MAV_MISSION_RESULT AP_Mission::mavlink_int_to_mission_cmd(const mavlink_mission_
     }
     break;
 
-    case MAV_CMD_NAV_LOITER_TIME:                       // MAV ID: 19
-        cmd.p1 = packet.param1;                         // loiter time in seconds uses all 16 bits, 8bit seconds is too small. No room for radius.
```

Comment:
Are we trying to change the meaning of this field?  The comment here says that all 16 bits are used in storage for time.

What happens to everybody who has a mission currently in storage on their vehicle when they move to this new firmware and just go for a fly?

Reply:
hmmm. yeah good point.

Is there any protocol for breaking changes in mission format? Are we allowed to break them e.g. on major versions or something?